Profile build broken

After 

https://github.com/official-stockfish/Stockfish/commit/fbb2ffacfdf10fc37d8ee2d2093b2cec629f6067

my profile build is broken

cardanobile@cardanobile-B75MA-P45:~/Dokumente/stefano/chess/gitProjects/Stockfish/src$ make profile-build ARCH=x86-64-modern && ./stockfish bench

Config:
debug: 'no'
sanitize: 'no'
optimize: 'yes'
arch: 'x86_64'
bits: '64'
kernel: 'Linux'
os: 'GNU/Linux'
prefetch: 'yes'
popcnt: 'yes'
sse: 'yes'
pext: 'no'

Flags:
CXX: g++
CXXFLAGS: -Wall -Wcast-qual -fno-exceptions -fno-rtti -std=c++11  -pedantic -Wextra -Wshadow -m64 -DNDEBUG -O3 -DIS_64BIT -msse -msse3 -mpopcnt -DUSE_POPCNT -flto
LDFLAGS:  -Wl,--no-as-needed -lpthread -Wall -Wcast-qual -fno-exceptions -fno-rtti -std=c++11  -pedantic -Wextra -Wshadow -m64 -DNDEBUG -O3 -DIS_64BIT -msse -msse3 -mpopcnt -DUSE_POPCNT -flto

Testing config sanity. If this fails, try 'make help' ...


Step 1/4. Building instrumented executable ...
make ARCH=x86-64-modern COMP=gcc gcc-profile-make
make[1]: Entering directory '/home/cardanobile/Dokumente/stefano/chess/gitProjects/Stockfish/src'
make ARCH=x86-64-modern COMP=gcc \
EXTRACXXFLAGS='-fprofile-generate'\
EXTRALDFLAGS='-lgcov' \
all
make[2]: Entering directory '/home/cardanobile/Dokumente/stefano/chess/gitProjects/Stockfish/src'
g++ -Wall -Wcast-qual -fno-exceptions -fno-rtti -std=c++11 -fprofile-generateEXTRALDFLAGS=-lgcov -pedantic -Wextra -Wshadow -m64 -DNDEBUG -O3 -DIS_64BIT -msse -msse3 -mpopcnt -DUSE_POPCNT -flto   -c -o benchmark.o benchmark.cpp
g++: error: unrecognized command line option ‘-fprofile-generateEXTRALDFLAGS=-lgcov’
<builtin>: recipe for target 'benchmark.o' failed
make[2]: *** [benchmark.o] Error 1
make[2]: Leaving directory '/home/cardanobile/Dokumente/stefano/chess/gitProjects/Stockfish/src'
Makefile:525: recipe for target 'gcc-profile-make' failed
make[1]: *** [gcc-profile-make] Error 2
make[1]: Leaving directory '/home/cardanobile/Dokumente/stefano/chess/gitProjects/Stockfish/src'
Makefile:429: recipe for target 'profile-build' failed
make: *** [profile-build] Error 2

and I suspect many of the machines on the framework have this problem too, see 

https://groups.google.com/forum/?fromgroups=#!topic/fishcooking/uTBn1ILLBlw

--- a/src/Makefile
+++ b/src/Makefile
@@ -523,7 +523,7 @@ clang-profile-use:
 
 gcc-profile-make:
 	$(MAKE) ARCH=$(ARCH) COMP=$(COMP) \
-	EXTRACXXFLAGS='-fprofile-generate'\
+	EXTRACXXFLAGS='-fprofile-generate' \
 	EXTRALDFLAGS='-lgcov' \
 	all
 

